config-overrides.js

```diff
@@ -1,18 +1,10 @@
 /* config-overrides.js */
 
 const path = require('path')
-const {
-  addWebpackAlias,
-  useBabelRc,
-  override,
-  useEslintRc,
-  enableEslintTypescript,
-} = require('customize-cra')
+const { addWebpackAlias, useBabelRc, override } = require('customize-cra')
 
 module.exports = override(
   useBabelRc(),
-  useEslintRc(path.resolve(__dirname, '.eslintrc.js')),
-  enableEslintTypescript(),
```

Comment:
We don't need to inject eslint separately because react-scripts migrated to `eslint-webpack-plugin` from v4.x on which reads the eslint config files from the root of the repository by default.